src/components/Navbar.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react'
+import { Dialog } from '@headlessui/react'
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+
+const navigation = [
+  { name: 'Services', href: '#services' },
+  { name: 'About', href: '#about' },
+  { name: 'Contact', href: '#contact' },
+]
+
+export default function Navbar() {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false)
+
+  return (
+    <header className="bg-white border-b border-gray-200 sticky top-0 z-50">
+      <nav
+        className="mx-auto flex max-w-7xl items-center justify-between p-4 lg:px-8"
+        aria-label="Top navigation"
+      >
+        {/* Left: Logo and title */}
+        <div className="flex items-center space-x-2">
+          <img src="/logo.png" alt="Logo" className="h-8 w-8" />
```

Comment:
Import the image file instead of linking to it - there are several reasons for this, but the main one is that the way it's currently written won't work when you deploy to production because the logo.png file won't be available in the `public/` folder. When you import the file, the bundler (vite) knows to include it in the bundle.